instrumentation/active_storage/opentelemetry-instrumentation-active_storage.gemspec

```
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/active_storage/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-active_storage'
+  spec.version     = OpenTelemetry::Instrumentation::ActiveStorage::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'ActiveStorage instrumentation for the OpenTelemetry framework'
+  spec.description = 'ActiveStorage instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = Dir.glob('lib/**/*.rb') +
+               Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 3.0'
+
+  spec.add_dependency 'opentelemetry-api', '~> 1.4.0'
+  spec.add_dependency 'opentelemetry-instrumentation-active_support', '~> 0.7'
+  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.22.6'
+
+  spec.add_development_dependency 'appraisal', '~> 2.5'
+  spec.add_development_dependency 'bundler', '~> 2.4'
+  spec.add_development_dependency 'minitest', '~> 5.0'
+  spec.add_development_dependency 'opentelemetry-sdk', '~> 1.1'
+  spec.add_development_dependency 'opentelemetry-test-helpers', '~> 0.3'
+  spec.add_development_dependency 'rake', '~> 13.0'
+  spec.add_development_dependency 'rubocop', '~> 1.69.1'
```

We're about to merge a PR that'll bump Rubocop to 1.70.0:
```suggestion
  spec.add_development_dependency 'rubocop', '~> 1.70.0'
```

---

Thank you for letting me know, I've matched the version with other packages!
https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1313/commits/b98fadfd952dcaa834725cd16dd2daf0f332eca1